src/App.js

```diff
@@ -85,17 +146,17 @@ handleSearch = async event => {
         <h1>Currecy</h1>
         <form>
           {/* <h1>This is what you search for: {this.state.searchCountry} </h1> */}
-         
+          <label for="searchCountry">Search:</label>
           <input type = 'text' name='searchCountry' onChange={this.handleSearch} placeholder='Search'/>
 
         </form>
         <table>
           <thead>
-          <tr>
-            <th>Country</th>
-            <th>BaseCurrency</th>
+          <tr class = "buttons">
+            <th><p onClick={this.handleCountry}> Country{arrows}</p></th>
+            <th><p onClick={this.handleBase} >BaseCurrency {arrows}</p></th>
```

Comment:
Let's try to have an "up-arrow" when sorting, and a "down-arrow" when the sort is reversed.